app/src/main/res/layout/fragment_camera.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".Fragments.CameraFragment">
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/camera_fragment"
```

Comment:
You are using underscore pattern or camelcase?

Reply:
undersscore